services/jobrunner/justfile

```diff
@@ -58,12 +59,31 @@ jobs-stats:
 
 # retry a specific job
 job-retry job_id:
-    python3 -m jobrunner.cli.retry_job {{ job_id }}
+    docker compose run jobrunner python3 -m jobrunner.cli.retry_job {{ job_id }}
+
+# kill a job
+kill-job *args:
+    docker compose run jobrunner python3 -m jobrunner.cli.kill_job {{ args }}
 
 # manually enable database maintenance mode. Kill and re-queue all db jobs.
 db-maintenance-on:
-    python3 -m jobrunner.cli.flags set manual-db-maintenance=on
+    docker compose run jobrunner python3 -m jobrunner.cli.flags set manual-db-maintenance=on
 
 # manually disable database maintenance mode
 db-maintenance-off:
-    python3 -m jobrunner.cli.flags set manual-db-maintenance=
+    docker compose run jobrunner python3 -m jobrunner.cli.flags set manual-db-maintenance=
+
+# update docker image
+update-docker-image image:
+    #!/bin/bash
+    set -euo pipefail
+    # pull from proxy 
+    docker pull docker-proxy.opensafely.org/opensafely-core/{{ image }}
+    # tag as if we'd pulled direct from ghcr.io 
+    docker tag docker-proxy.opensafely.org/opensafely-core/{{ image }} ghcr.io/opensafely-core/{{ image }}
+    docker image prune --force
```

Comment:
aside: when I was updating the ehrql docker image yesterday, it pruned some airlock images - which was unexpected! but not /incorrect/ as such...